client/bridge_client.go

```diff
@@ -172,6 +172,34 @@ func (ec *Client) BridgeGetSentChainTxsLimit(ctx context.Context) (uint64, error
 	return result, err
 }
 
+// BridgeRegisteredOperators can register an operator.
+func (ec *Client) BridgeRegisterOperator(ctx context.Context, bridge, operator common.Address) (common.Hash, error) {
+	var result common.Hash
+	err := ec.c.CallContext(ctx, &result, "subbridge_registerOperator", bridge, operator)
+	return result, err
+}
+
+// BridgeGetRegisteredOperators can return the list of operators.
+func (ec *Client) BridgeGetRegisteredOperators(ctx context.Context, bridge common.Address) (uint8, error) {
+	var result uint8
```

Comment:
why is the type of result uint8?
Doesn't it return the length of the list of operators?
then could you please change the comments?? :)

Reply:
Oops... My mistake.

Reply:
fixed :)